supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -874,6 +885,16 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rolling-stream]]
+==== image:images/yes.png[yes] Rolling stream (noun)
+*Description*: _Rolling Application Streams_ include RHEL tools and applications that move very fast and are updated frequently. Later versions of these streams replace versions released earlier. Therefore, only one version is supported at a time.
```

Comment:
```suggestion
*Description*: _Rolling Application Streams_ include RHEL tools and applications that are updated frequently. Later versions of these streams replace versions released earlier. Therefore, only one version is supported at a time.
```
- Omit `move very fast` because tools and applications can't actually move.